src/WSServerCon.cpp

```diff
@@ -225,7 +225,7 @@ void WSServerCon::processMessage(const QString &message)
     {
         QJsonObject o = root["data"].toObject();
         QString loginName = o["login"].toString();
-        if (loginName.isEmpty() && !o.contains("saveConfirmed"))
+        if (loginName.isEmpty() && o.contains("extension_version") && !o.contains("saveConfirmed"))
```

Comment:
I'm against this. First it does not belong to this PR, second if we use the `mc-cli` or the API to set an empty login we do not get the prompt.

Reply:
first point: agreed, though we eventually need to integrate it somewhere
second point: I had forgotten about mc-cli. Could we add a "mc_cli_version" key and do a OR on that condition?